docs/user/03-30-hap-assignment.md

```diff
@@ -0,0 +1,31 @@
+
+## IaaS Provider Account Assignment Rules
+
+The specific assignment options are the following:
+
+- Amazon Web Services offers the following account pools:
+  - An account pool for Kyma instances created with the standard service plans `aws`, `build-runtime-aws`, and the `free` plan
+  - An account pool for Kyma instances created with the standard service plans `aws` and `build-runtime-aws` in the subaccount <!--platform?--> region cf-eu11, Europe (Frankfurt) EU Access
+  - A pool of shared accounts for trial Kyma instances
+
+- Microsoft Azure offers the following account pools:
+  - An account pool for Kyma instances created with the standard service plans `azure`, `build-runtime-azure`, test demo and development plan `azure-lite`, and the `free` plan
+  - An account pool for Kyma instances created with the standard service plans `azure`, `build-runtime-azure` in the subaccount <!--platform?--> region cf-ch20, Switzerland (Zurich) EU Access
+
+- Google Cloud offers the following account pools:
+  - An account pool for Kyma instances created with the standard service plans `gcp` and `build-runtime-gcp`
+  - An account pool for Kyma instances created with the standard service plans `gcp` and `build-runtime-gcp` in the subaccount <!--platform?--> region cf-sa30, KSA (Dammam) GCP public sector, requiring Assured Workloads Kingdom of Saudi Arabia (KSA) control package.
+
+- <!--INTERNAL! Check this!!!!--> SAP Cloud Infrastructure offers one pool of shared accounts for each of the cluster regions available for the service plan `sap-converged-cloud`
+
+
+For example, once you create the first Kyma instance with the standard `aws` plan in a non-EU region in your global account, it is automatically assigned to an account provided by Amazon Web Services. After that, all your Kyma instances in all subaccounts within the same global account created with the `aws` or `build-runtime-aws` plans in non-EU regions are assigned to the same Amazon Web Services account. Suppose you have no Kyma instances created in the `aws` EU region in the same global account. In that case, this is your only assignment to an Amazon Web Services account, regardless of the number of existing Kyma instances with the standard `aws` plan. However, if, in the same global account, there are also Kyma instances created with the `aws` or `build-runtime-aws` plans in the `aws` EU region, they are all assigned to another single account provided by Amazon Web Services. Therefore, your Kyma instances in this particular global account are assigned to two IaaS provider accounts. Adding more Kyma instances with other plans results in more assignments to the corresponding IaaS provider accounts.
+
+## Migrating Assigned Kyma Instances
```

Comment:
Maybe we should find better name for this chapter. More saying about BTP feature, that users can move SA to another GA